multicluster/controllers/multicluster/common/helper.go

```diff
@@ -52,3 +54,21 @@ func RemoveString(slice []string, s string) (result []string) {
 	}
 	return
 }
+
+// FilterSubsets keeps IPs only and remove others which are unnecessary information for other member cluster.
+func FilterSubsets(subsets []corev1.EndpointSubset) []corev1.EndpointSubset {
```

Comment:
Name it FilterEndpointSubsets to be clearer

Reply:
done